accname/name/comp_label.html

```diff
@@ -11,11 +11,140 @@
+<h4 aria-label="label" data-expectedlabel="label" data-testname="label valid on h4 element" class="ex">x</h4>
+<h5 aria-label="label" data-expectedlabel="label" data-testname="label valid on h5 element" class="ex">x</h5>
+<h6 aria-label="label" data-expectedlabel="label" data-testname="label valid on h6 element" class="ex">x</h6>
+<img alt="" aria-label="label" data-expectedlabel="label" data-testname="label valid on img element" class="ex" />
+<a href="" aria-label="label" data-expectedlabel="label" data-testname="label valid on link element" class="ex">x</a>
+<ul aria-label="label" data-expectedlabel="label" data-testname="label valid on list (unordered) element" class="ex">x</ul>
+<ol aria-label="label" data-expectedlabel="label" data-testname="label valid on list (ordered) element" class="ex">x</ol>
+<li aria-label="label" data-expectedlabel="label" data-testname="label valid on listitem element" class="ex">x</li>
+<main aria-label="label" data-expectedlabel="label" data-testname="label valid on main element" class="ex">x</main>
+<math aria-label="label" data-expectedlabel="label" data-testname="label valid on math element" class="ex">x</math>
+<meter aria-label="label" data-expectedlabel="label" data-testname="label valid on meter element" class="ex">x</meter>
+<nav aria-label="label" data-expectedlabel="label" data-testname="label valid on nav element" class="ex">x</nav>
+
+<!-- <option aria-label="label" data-expectedlabel="label" data-testname="label valid on option element" class="ex">x</option> -->
+<input type="radio" aria-label="label" data-expectedlabel="label" data-testname="label valid on input type radio element" class="ex" />
+<section aria-label="label" data-expectedlabel="label" data-testname="label valid on section element" class="ex">x</section>
+<tr aria-label="label" data-expectedlabel="label" data-testname="label valid on tr element" class="ex">x</tr>
+<tbody aria-label="label" data-expectedlabel="label" data-testname="label valid on tbody element" class="ex">x</tbody>
+<tfoot aria-label="label" data-expectedlabel="label" data-testname="label valid on tfoot element" class="ex">x</tfoot>
+<thead aria-label="label" data-expectedlabel="label" data-testname="label valid on thead element" class="ex">x</thead>
+<th scope="row" aria-label="label" data-expectedlabel="label" data-testname="label valid on th (scope row) element" class="ex">x</th>
+<input type="search" aria-label="label" data-expectedlabel="label" data-testname="label valid on input type search element" class="ex" />
+<hr aria-label="label" data-expectedlabel="label" data-testname="label valid on hr element" class="ex" />
+<input type="text" aria-label="label" data-expectedlabel="label" data-testname="label valid on input type text element" class="ex" />
+<textarea aria-label="label" data-expectedlabel="label" data-testname="label valid on textarea element" class="ex">x</textarea>
+
+
+<!-- TODO write tests that purposefully fail?? -->
```

Comment:
Clarifying that I meant ASSERT_FALSE.

Reply:
@rahimabdi pointed out there is also ASSERT_NOT_EQUALS 👍🏼

Reply:
Added `assertAriaLabelIsIgnored()` as part of c0f18223279b099c1d6c16a68.

(I had trouble coming up with a clear function name due to the desire to avoid negative function names.g., NameNotEqualsAriaLabel; if you have a better name that uses the affirmative, please let me know!)

Reply:
Per @jugglinmike's recommendation, updated `assertAriaLabelIsIgnored()` to `verifyAriaLabelIsIgnored()` as part of bb8739ec77333e6ff58e7ad67ab462e5391bd16e.